HelpSource/Classes/Server.schelp

```diff
@@ -244,6 +249,47 @@ argument:: limit
 The number of times to check for a successful boot.
 argument:: onFailure
 A function to evaluate after the server fails to boot. If onFailure is not given, an error message is posted. Providing a function suppresses the error message. If you want to supply a function and print the normal error message, make sure that your function returns "false," e.g. code::s.doWhenBooted(onFailure: { ... custom action...; false })::.
+argument:: clock
+The default link::Classes/Clock:: is link::Classes/AppClock::. Use link::Classes/SystemClock:: or link::Classes/TempoClock:: if time accuracy is important.
+
+discussion::
```

Comment:
This sounds as if this were a feature of doWhenBooted, while it is a standard syntax. Maybe the explanation can be left out? It is good, but complicates things.

Reply:
Thanks!
I replaced the explanation
>The default link::Classes/Clock:: is link::Classes/AppClock::. Use link::Classes/SystemClock:: or link::Classes/TempoClock:: if time accuracy is important.

with the following one sentence:
>If left empty, link::Classes/AppClock:: will be used.
How about this?

https://github.com/supercollider/supercollider/pull/6198/commits/e98caae4112ac274c3ddc08b3c76814600abe844

Reply:
This is a misunderstanding, sorry. I meant: "When using the default clock of code::.waitForBoot:: and code::.doWhenBooted::, the pair of braces surrounding the curly bracket can be omitted:"

What you removed is good specific information, I think. And below, we refer to it ("(see above)").

Reply:
Is it now ok?
https://github.com/supercollider/supercollider/pull/6198/commits/41a766beb7c3f165c4541bd05a877bcf16fe60ae